src/style.css

```diff
@@ -0,0 +1,153 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+  font-style: normal;
+}
+
+html {
+  font-family: Avenir, Helvetica, sans-serif;
+  font-weight: 300;
+  font-size: 16;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.formBig {
+  display: block;
+  box-sizing: border-box;
+  width: 100%;
+  margin-top: 20px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  height: 70px;
+  position: relative;
+}
+
+.bigSearch {
+  border: none;
+  outline: none;
+  width: 100%;
+  height: 100%;
+  background-color: transparent;
+  font-family: inherit;
+  font-size: inherit;
+  line-height: 22px;
+  font-weight: 800;
+  color: #3d4e61;
+}
+
+.bigS {
+  position: absolute;
```

Comment:
![Screenshot from 2023-02-05 13-15-35](https://user-images.githubusercontent.com/33029864/216815984-1f46c413-b1bf-470c-b360-15e0d8233f8b.png)
input not fill on all width
to fix it add widht: 100% to label

but pay attention after that to another problem
![Screenshot from 2023-02-05 13-20-04](https://user-images.githubusercontent.com/33029864/216816025-6ed7aaa5-165a-45a0-a27a-7faa0db53345.png)

this problem in position: absolote and margin-left for label

Reply:
to fix it rethink about structure of form
instead of form give position relative for label and replace pseudo element from form to label

Reply:
ready